plugins/action/stop_region.py

```diff
@@ -173,8 +143,46 @@ def _get_job_status(self):
         if job_status == "COMBINATION INVALID":
             raise AnsibleActionFail(
                 "No jobs found with name {0} and ID {1}".format(self.job_name, self.job_id))
+
         return job_status
 
+    def _get_job_id_and_status_by_name(self):  # type: () -> (str, str)
+        # If we have a name but no ID, we use a TSO command to get the job ID
+        running_jobs = self._get_running_jobs()
+
+        if len(running_jobs) == 0:
+            # In the event that the job ID is missing, we're still not exposed to the ZOAU 'bug' as we have not used zos_job_query
+            self.job_status = "MISSING"
+            return
```

Comment:
Think I've just hit a bug with this rouge return here. If we do a none type return then we cant unpack the tuple when this function gets called